### PR TITLE
Fixed broken build

### PR DESCRIPTION
**Pull request by Nyholm**
This PR updates travis.yml to adapt to our best practices: https://symfony.com/doc/current/bundles/best_practices.html

It update our dependencies (dropping sf2.8 and unsupported symfony versions)
It also remove use of deprecated code.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Should we bump to v2 then?

**Comment by Nyholm** [comment]
Yes, And introduce PSR17 factories. The PSR17 does not support php5.

**Comment by Nyholm** [comment]
I just had a look at #43. That implementation makes it optional to use PSR 17 (HTTP factories). Then it makes no sense to drop PHP versions.

This PR brings no changes nor drops any functionality.

**Comment by Nyholm** [comment]
Wohoo build passes on Travis!

**Comment by Nyholm** [comment]
Ping @nicolas-grekas

File: .travis.yml [hunk 1]
```diff
@@ -1,49 +1,66 @@
...
-env:
-    global:
-        - deps=no
-        - SYMFONY_DEPRECATIONS_HELPER=strict
-        - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
-        - COMPOSER_OPTIONS="--prefer-stable"
+          # Test LTS versions. This makes sure we do not use Symfony packages with version greater
+          # than 2 or 3 respectively.
+        - php: 7.2
+          env: DEPENDENCIES="symfony/lts:^2 symfony/force-lowest:^2.8 zendframework/zend-diactoros:^1.4.1"
```
Comment (nicolas-grekas) [inline .travis.yml]: symfony/force-lowest should reference an exact version to work correctly:
symfony/force-lowest:=2.8 or symfony/force-lowest:v2.8

**Comment by Nyholm** [comment]
Travis is happy again

**Comment by fabpot** [comment]
Thank you @Nyholm.

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**